tests/foreman/ui/test_isodownload.py

```diff
@@ -0,0 +1,184 @@
+from robottelo.test import UITestCase
+
+
+class TestISODownloads(UITestCase):
+    """Test class for iso download feature"""
+
+    def test_iso_download_1(self):
+        """@test: Downloading iso's from export
+
+        @Feature: ISO Downloads
+
+        @Setup: ISO Downloads
```

Comment:
This setup does not says much, if there is no special setup it could be dropped. Otherwise it should be expanded.